engine/class_modules/sc_rogue.cpp

```diff
@@ -7878,8 +7979,11 @@ void actions::rogue_action_t<Base>::trigger_seal_fate( const action_state_t* sta
   if ( !p()->rng().roll( p()->talent.assassination.seal_fate->effectN( 1 ).percent() ) )
     return;
 
-  trigger_combo_point_gain( as<int>( p()->talent.assassination.seal_fate->effectN( 2 ).trigger()->effectN( 1 ).base_value() ),
-                            p()->gains.seal_fate );
+  auto gain = as<int>( p()->talent.assassination.seal_fate->effectN( 2 ).trigger()->effectN( 1 ).base_value() );
+  if ( procs_deal_fate() && p()->talent.fatebound.deal_fate->ok() ) {
+    gain += as<int>( p()->talent.fatebound.deal_fate->effectN( 1 ).base_value() );
```

Comment:
Reminder for myself: this should probably be a secondary `trigger_combo_point_gain` call, so the gain is tracked separately from `Seal Fate` in the stats, like it is for outlaw's version.